docker-compose/install.sh

```diff
@@ -8,18 +8,18 @@ install_sunbird_rc() {
   cd ..
 }
 
-install_esignet() {
```

Comment:
Why "loader_path/mock-esignet-integration-impl-0.9.2-SNAPSHOT.jar" is required ?

Reply:
the code is not seggregated in esignet and is still dependent on mock-esignet-integration-impl for LoggerAudit service. So this is needed for running esignet service